storage/google/cloud/storage/blob.py

```diff
@@ -1911,6 +1911,7 @@ def metadata(self, value):
         :type value: dict
         :param value: (Optional) The blob metadata to set.
         """
+        value = {k: str(v) for k, v in value.items()}
```

Comment:
Thanks @HemangChothani!

Could you also add this to bucket labels given the similarity?